tests/test_client/tests.py

```diff
@@ -278,6 +287,37 @@ def test_redirect_http(self):
         response = self.client.get('/http_redirect_view/', follow=True)
         self.assertFalse(response.test_was_secure_request)
 
+    def test_assert_url_equal(self):
```

Comment:
These tests are fine. I just wonder if they shouldn't live in `tests/test_utils` somewhere?